code/modules/mob/living/carbon/human/life.dm

```diff
@@ -136,21 +135,19 @@
 							   "lifelike texture",
 							   "luv can bloooom",
 							   "PACKETS!!!",
-							   "[pick("WHERE MY","aYE need","giv me my","bath me inn.")] [pick("dermaline","alKkyZine","dylOvene","inAprovaline","biCaridine","Hyperzine","kELotane","lePorazine","bAcch Salts","tricord","clOnexazone","hydroChloric Acid","chlorine Hydrate","paRoxetine")]!",
+							   "[pick("WHERE MY","aYE need","giv me my","bath me inn.")] [pick("dermaline","alKkyZine","dylOvene","inAprovaline","biCaridine","Hyperzine","kELotane","lePorazine","bAcch Salts","tricord","clOnexazone","hydroChloric Acid","chlORine Hydrate","paRoxetine")]!",
 							   "mALPRACTICEBAY",
 							   "I HavE A pe H dee iN ENTerpriSE resOUrCE pLaNNIN",
 							   "h-h-HalP MaINT",
 							   "dey come, dey COME! DEY COME!!!",
 							   "THE END IS NIGH!",
-							   "I FOT AND DIED FOR MUH [pick("RITES","FREEDOM","payCHECK","cARGO points","teCH Level","doG","mAPLe syrup","fluffy fWiends","gateway Loot")]",
-							   "KILL DEM [pick("mainTnacE cHickinNS","kiRA CulwnNES","FLOOR CLUWNEs","MIME ASSASSIN","BOMBING TAJARAN","cC offiser","morPhlings","slinglings")]!")
-			switch(pick(1,2,3))
-				if(1)
-					say(pick(s1))
-				if(2)
-					say(pick(s2))
-				if(3)
-					emote("drool")
+							   "I FOT AND DIED FOR MUH [pick("RITES","FREEDOM","payCHECK","cARGO points","teCH Level","doG","mAPLe syrup","fluffy fWiends","gATEway LoOt")]",
+							   "KILL DEM [pick("mainTnacE cHickinNS","kiRA CulwnNES","FLOOR CLUWNEs","MIME ASSASSIN","BOMBING TAJARAN","cC offiser","morPhlings","slinglings")]!",
+							   "I CAN FORCE YOU TO SAY WHATEREVE!!?!?!")
+			if(prob(66))
+				say(pick(crazysay))
+			else
+				emote("drool")
```

Comment:
It might be worth moving the drool emote outside of this block and giving it its own probability to fire. It'd be nice to have this as a more visible tell for brain damage, and let the prob(2) just fire these messages off on its own.

Reply:
I tried it and it was kinda clunky, because you sometimes got drooling and saying stuff at the same time which feels kinda spammy. I could increase the chance on drooling, move it to the front of the proc and make it return, and then increase it from prob(3) to maybe something like prob(5)